src/views/Gene/Template.vue

```diff
@@ -1,3 +1,6 @@
+
+import ResearchExpressionPlot from '../../components/researchPortal/ResearchExpressionPlot.vue';
```

Comment:
another way to link is to start with **_@_** which is equivalent to **_src_** folder, if you find yourself linking from too many levels _../.._

Reply:
Done.